dashboard/app/controllers/pd/workshop_dashboard_controller.rb

```diff
@@ -4,13 +4,15 @@ class WorkshopDashboardController < ApplicationController
     def index
       @permission = nil
 
+      contact_rp = RegionalPartner.find_by(contact: current_user)
```

Comment:
Indeed! This is just for the permission check; we collect the other associations [below](https://github.com/code-dot-org/code-dot-org/pull/13647/files#diff-73918c02eb1a048f209a8dd8c0965d11R24)